### DISCUSSION TITLE
ww3_multi input files

### DISCUSSION THREAD

**Discussion by fed1942** [NOAA-EMC/WW3#953] [Q&A] [answered]
Does anyone know how to generate mod_def.input and wind.input in WaveWatch III? I am using 2 grids: Global (coarse) and Regional (refined). For each of the grids i have mod_def.ww3, restart.ww3 and wind.ww3. I read that you have to replace the "ww3" extension with the "input" extension. The problem is that i have two mod_def.ww3 files and two wind.ww3 files.

Thank you in advance.

**Comment by JessicaMeixner-NOAA** [comment, marked as answer]
So it sounds like you want to run a multi-grid case? If so, you want to rename mod_def.ww3 -> mod_def.coarse and restart.corse, etc for the coarse grid and then mod_def.ww3 -> mod_def.regional,  wind.ww3 -> wind.regional where 'coarse' and 'regional' can be replaced by any name as long as it's consistent and then you use those names in your ww3_multi.inp or ww3_multi.nml files.

**Reply by fed1942** [reply]
Yes, i want to run a multi-grid case. The question is: which one of the mod_def files (mod_def.Coarse or mod_def.Regional) should be renamed as mod_def.input?

**Reply by JessicaMeixner-NOAA** [reply]
If you have two wind.ww3 files, it seems like you don't need an input grid.  Instead you can just say: 
```
  'coarse'  'no' 'no' 'native' 'no' 'no' 'no' 'no' 'no' 'no'  1  1  0.00 1.00  F
  'regional'  'no' 'no' 'native' 'no' 'no' 'no' 'no' 'no' 'no'  2  1  0.00 1.00  F
```

**Reply by fed1942** [reply]
Now i think i understand. So, for example, in the first line of the multi.inp file i should put 2 0 F 1 T T?

**Reply by JessicaMeixner-NOAA** [reply]
The 2 wave model grids and the 0 input grids is correct based on the information you've shared.
